src/Models.jl

```diff
@@ -37,11 +37,15 @@ Convention is that `weights` defaults to `StatsBase.uweights(Float32, size(outpu
 function fit end
 
 """
-    predict(::Model, input)
+    predict(model::Model, inputs::AbstractMatrix)
 
-Predict targets for the provided `input` and [`Model`](@ref).
+Predict targets for the provided the collection of `inputs` and [`Model`](@ref).
 
-Returns a predictive distribution or point estimates depending on the [`Model`](@ref).
+If the `estimate_type(model) == [`PointEstimate`](@ref) then this function should return
```

Comment:
to make back-ticks happy

```suggestion
If the `estimate_type(model)` is [`PointEstimate`](@ref) then this function should return
```